### PR TITLE
Update part-6-performance-normalization.md

### PR DESCRIPTION
**Pull request by aleksanderantropov**
To make it consistent with the previous example

Thanks for the PR!

To better assist you, please select the type of PR you want to create.

Click the "Preview" tab above, and click on the link for the PR type:

- [:bug: Bug fix or new feature](?template=bugfix.md)
- [:memo: Documentation Fix](?template=documentation-edit.md)
- [:book: New/Updated Documentation Content](?template=documentation-new.md)

### CODE REVIEW

**Comment by markerikson** [comment]
Yeah, looks like I did forget to update the `notificationsSlice` one.

I'll reopen this - can you change the PR to fix that one instead?

File: docs/tutorials/essentials/part-6-performance-normalization.md [hunk 1]
```diff
@@ -362,17 +363,18 @@ const notificationsSlice = createSlice({
...
-      state.forEach(notification => {
-        // Any notifications we've read are no longer new
-        notification.isNew = !notification.read
-      })
-      // highlight-end
-      // Sort with newest first
-      state.sort((a, b) => b.date.localeCompare(a.date))
-    })
+    builder
+      .addCase(fetchPosts.fulfilled, (state, action) => {
```
Comment (markerikson) [inline docs/tutorials/essentials/part-6-performance-normalization.md:367]: :hand: This just changed from `fetchNotifications.fulfilled` to `fetchPosts.fulfilled`, which doesn't look right.

**Review by 3imed-jaberi — APPROVED** [review]
+1

**Review by sokarax — APPROVED** [review]

**Comment by markerikson** [comment]
I apparently updated this elsewhere at some point.

**Pull request closed without merging**